automated-actions/AWS_RISK_CREDENTIALS_EXPOSED/README.md

```diff
@@ -0,0 +1,173 @@
+# AWS Health AWS_RISK_CREDENTIALS_EXPOSED
+
+This example demonstrates how to use [AWS Step Functions](https://aws.amazon.com/step-functions/) to orchestrate a serverless [AWS Lambda](http://aws.amazon.com/lambda/) workflow in response to an [Amazon CloudWatch](https://aws.amazon.com/cloudwatch/) Event generated by [AWS Health](http://docs.aws.amazon.com/health/latest/ug/what-is-aws-health.html). 
+
+AWS proactively monitors popular code repository sites for exposed IAM access keys. On detection of an exposed IAM access key, AWS Health generates an AWS_RISK_CREDENTIALS_EXPOSED CloudWatch Event. In response to this event, an automated workflow deletes the exposed IAM Access Key, summarizes the recent API activity for the exposed key, and sends the summary message to an SNS Topic to notify the subscribers- all orchestrated by an AWS Step Functions state machine.
```

Comment:
Fixed in this commit: https://github.com/aws/aws-health-tools/pull/26/commits/baafc53b97e85990c5ad49ea9e32f69f0e926688